Hudl.Mjolnir/Config/StaticBulkheadConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Hudl.Mjolnir.Config
+{
+    /// <summary>
+    /// Used only for configs which will never change.
```

Comment:
Minor, but is this true? If observables aren't used, configuration values can still change, correct? It's just that they won't fire change events. A re-read of the configuration property may yield a different value, though, right?

Reply:
In that case "Static" may be a bit misleading as the type name here. Could go with something like `NonObservableBulkheadConfiguration`?

Reply:
For the first comment: It will never change since this is only used for DefaultMjolnirConfig which is implemented inside the library already. If someone will provide different config they have to implement their own implementation of BulkheadConfiguration. 
I'll make that class internal! 
Also I'll change it's name.